index.html

```diff
@@ -5,27 +5,136 @@
 		<link type="text/css" rel="stylesheet" href="styles.css">
```

Comment:
```suggestion
		<link type="text/css" rel="stylesheet" href="https://trunk-mysql.nightly.sakaiproject.org/library/skin/morpheus-default/properties.css">
		<link type="text/css" rel="stylesheet" href="styles.css">
```

Do you want to bring in the CSS props from master so we keep them in sync?

Reply:
Yes, I was thinking about utilizing the properties, but wasn't quite ready for them yet in my design, so I just hand-picked them for now. I'll include them in a future PR.